docs/make-all-versions.sh

```
@@ -0,0 +1,92 @@
+                   "v0\.5\.0rc[0-9]+")
+
+# find the current branch name
+CURRENT_BRANCH=$(git rev-parse --abbrev-ref HEAD)
+
+# This script is used to generate all versions of the documentation.
+rm -rf docs/build_all 2>/dev/null || true
+mkdir -p docs/build_all
+RELEASES=$(git tag)
+
+# remove excluded releases
+for release in ${EXCLUDED_RELEASES[@]}; do
```

Shellcheck (0.10.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)